File: app/src/main/java/com/killain/organizer/packages/interfaces/FragmentUIHandler.java
package com.killain.organizer.packages.interfaces;

import android.view.View;

import com.killain.organizer.packages.enums.AdapterRefreshType;

public interface FragmentUIHandler {
    View getBackground();
    void refreshAdapterOnAdd(int position, AdapterRefreshType refreshType);
    void refreshAdapterOnDelete(int position);
}

File: app/src/main/java/com/killain/organizer/packages/interfaces/FragmentUIHandler.kt
package com.killain.organizer.packages.interfaces

import android.view.View

import com.killain.organizer.packages.enums.AdapterRefreshType

interface FragmentUIHandler {
    val background: View?
    fun refreshAdapterOnAdd(position: Int, refreshType: AdapterRefreshType)
    fun refreshAdapterOnDelete(position: Int)
}
